src/components/InterestForm/index.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import { useState } from 'react';
+import { UUID } from 'crypto';
+import { insertInterest } from '../../api/supabase/queries/interest';
+import { Interest, CaseListing } from '../../types/schema';
+import { Button } from '../Button';
+import {
+  FormContainer,
+  FormTextArea,
+  FormInput,
+  RadioGroup,
+  RadioLabel,
+  RadioInput,
+} from './styles';
+
+const radioOptions = ['Attorney', 'Interpreter', 'Both Attorney & Interpreter'];
+
+export default function InterestForm({ caseData }: { caseData: CaseListing }) {
+  const [reason, setReason] = useState<string>('');
+  const [rolesInterested, setRolesInterested] = useState<string>('');
+  const [startDate, setStartDate] = useState<string>('');
+
+  const handleInsert = async () => {
+    if (reason !== '') {
+      const newInterest: Interest = {
+        // hardcoded values for now
+        id: crypto.randomUUID() as UUID,
+        listing_id: caseData.id,
+        listing_type: 0,
+        user_id: '36b8f84d-df4e-4d49-b662-bcde71a8764f',
+        form_response: {
+          interestType: 'IJP-6-test',
+          interestReason: reason,
+          start_date: startDate,
+        },
+      };
+      await insertInterest(newInterest);
+      setReason('');
+    } else {
+      // console.log('cannot create an empty interest');
+    }
+  };
+
+  return (
+    <FormContainer>
+      {/* <input type="text" /> */}
+      <strong>Submit Interest</strong>
+      <div>What role(s) are you applying for?</div>
```

Comment:
Is there a reason why "What role(s) are you applying for?" is a <div>, and the other questions are <p>?

I think the text needs different styling (the font color + weight doesn't match the figma). 
Maybe we should create a custom styled P component in a future sprint → use the custom P component for the question texts. 
@davidqing6432 thoughts?

Reply:
Catherine your comments are incredible!! Thanks for your attention to detail :))

Reply:
thank you frfrf, I should've been adding more things to the global styles to begin with lol 😅 handled it now and will be pushing all the fixes soon!